sycl/doc/extensions/experimental/sycl_ext_adaptivecpp_enqueue_custom_operation.asciidoc

```diff
@@ -0,0 +1,44 @@
+# sycl_ext_adaptivecpp_enqueue_custom_operation
+
+## Notice
+
+This document describes an **experimental** API that applications can use to try
+out a new feature. Future versions of this API may change in ways that are
+incompatible with this experimental version.
+
+## Introduction
+
+This extension is an experimental AdaptiveCpp extension, which is documented
+[here](https://github.com/AdaptiveCpp/AdaptiveCpp/blob/develop/doc/enqueue-custom-operation.md)
+
+## Feature test macro
+
+As encouraged by the SYCL specification, a feature-test macro,
+`SYCL_EXT_ACPP_ENQUEUE_CUSTOM_OPERATION`, is provided to determine
```

Comment:
For historic reasons, our feature test macro is called `ACPP_EXT_ENQUEUE_CUSTOM_OPERATION`. 
https://github.com/AdaptiveCpp/AdaptiveCpp/blob/2636e1ffb2ea4ceffb20610d7783d3163320c67a/include/hipSYCL/sycl/extensions.hpp#L71

It might be a good idea for AdaptiveCpp to move to what the SYCL spec encourages as a pattern, but it might also be worth consideration whether you want to support the AdaptiveCpp feature test macro in addition, for the benefit of existing user code.